dms/security/security.xml

```diff
@@ -67,4 +68,30 @@
         <field name="perm_unlink" eval="1" />
         <field name="domain_force">[(1 ,'=', 1)]</field>
     </record>
+    <record id="rule_security_dms_directory" model="ir.rule">
+        <field name="name">Dms Directories with dms group</field>
```

Comment:
I'm not very comfortable exposing this via UI, so please put in the name of both that this will be filtered later by code.